### PR TITLE
Typo fix

### PR DESCRIPTION
**Pull request by meghanmc04**

### CODE REVIEW

**Comment by meghanmc04** [comment]
REVIEW 10/10 CHOOSE ME

File: README.md [hunk 1]
```diff
@@ -139,6 +139,10 @@ This will show the same thing as before but will indicate your file is ready to
 8. Commit your change.  
 `git commit -m "Your commit message"  
 
+
+Nick Cannon wasn't here
```
Comment (jjallen37) [inline README.md:143]: No he wasn't

File: README.md [hunk 2]
```diff
@@ -139,6 +139,10 @@ This will show the same thing as before but will indicate your file is ready to
 8. Commit your change.  
 `git commit -m "Your commit message"  
 
+
+Nick Cannon wasn't here
+
```
Comment (meghanmc04) [inline README.md:144]: but like sorry he was

**Review by jjallen37 — COMMENTED** [review]
Ok fine Nick Cannon was here I guess. Pretty sure he's like a war criminal or something now

**Review by jjallen37 — APPROVED** [review]
Approved

**Pull request merged**